src/PKSim.Core/Services/RegisterObjectVisitor.cs

```diff
@@ -45,12 +42,5 @@ public void Visit(IObjectBase objToVisit)
          _withIdRepository.Unregister(objToVisit.Id);
          register(objToVisit);
       }
-
-      public void Visit(Simulation simulation)
```

Comment:
@Yuri05 This is not necessary. The simulation is already performing the iterations. In essensce, all used building block where registered twice. Not a problem because register of object base did unregister but time consuming for nothing